module/User/src/User/Mapper/User.php

```diff
@@ -60,11 +60,11 @@ public function findByLogin($login)
         if (is_numeric($login)) {
             $qb->where('u.lidnr = ?1');
         } else {
-            $qb->where('m.email = ?1');
+            $qb->where('m.email LIKE ?1');
         }
 
         // set the parameters
-        $qb->setParameter(1, $login);
+        $qb->setParameter(1, $login = str_replace("%", "", $login));
```

Comment:
I think the `$login = ` doesn't quite belong here. Also, single quotes > double quotes.